static/src/javascripts/projects/common/modules/ui/cookiesBanner.js

```diff
@@ -33,9 +43,20 @@ const init = (): boolean => {
     const msg = new Message('cookies', opts);
     msg.show(txt);
     addCookie(EU_COOKIE_MSG, 'seen', cookieLifeDays);
-    return true;
 };
 
+const init = (): void => {
+    canShow().then(result => {
+        if (result) {
+            show();
+        }
+    });
+};
+
+export { init };
```

Comment:
Is it worth adding a `// TODO` comment here to indicate your intention to delete this named export later? Or are you going to do this pretty soon (god willing) and it's not worth flagging?